cql3/statements/modification_statement.cc

```diff
@@ -439,13 +442,31 @@ modification_statement::process_where_clause(data_dictionary::database db, expr:
     }
 }
 
+::shared_ptr<strongly_consistent_modification_statement>
+modification_statement::prepare_for_raft() const {
+    // FIXME: implement for every type of `modification_statement`.
```

Comment:
This should be "prepare_for_broadcast_tables". Raft is just a tool here.

Reply:
Done.